app/Console/Kernel.php

```diff
@@ -28,6 +28,11 @@ protected function schedule(Schedule $schedule)
             ->environments(['staging', 'dev', 'local']) // only run for APP_ENV tagged staging, dev, or local
             ->onOneServer(); // run only on a single server at once
 
+        $schedule->command('migrate:fresh --seeder=DevSeeder') // use custom command to make sure that the commands are chained
+            ->dailyAt('00:15') // Run daily at 12:15 am
```

Comment:
Do you know which timezone this is?

Reply:
What timezone would you like and I will declare it explicitly. I am sure it probably uses server time by default.

Reply:
Done.